### DISCUSSION TITLE
How to do the input box auto line feed?

### DISCUSSION THREAD

**Discussion by Arbureva** [AllenDang/giu#563] [Q&A]
The input component has a maximum width of 300, but in actual testing the characters I enter cannot do automatic line feeds, and due to the appearance of complex characters such as emoticons, achieving automatic line feeds may become a little more complicated, does giu have this feature?

**Comment by Seikaijyu** [comment]
I haven't seen the relevant mark : D

**Reply by Seikaijyu** [reply]
I also hope to have this function

**Comment by AllenDang** [comment]
Do you mean auto wrap？

**Reply by Arbureva** [reply]
yes

**Reply by gucio321** [reply]
@Locter9001 see #434
